src/translate.component.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+  Sanitizer,
+  SecurityContext,
+  SimpleChange
+} from '@angular/core';
+import {
+  LangChangeEvent,
+  TranslateService,
+  TranslationChangeEvent
+} from './translate.service';
+
+@Component({
+  selector: '[translate]',
+  template: '<ng-content></ng-content>'
+})
+export class TranslateComponent implements OnInit, OnChanges, OnDestroy {
+  @Input('translate')
```

Comment:
It's bad practice to rename the bindings like this. Why not `@Input() translate: string` directly?